addons/account/models/account_move.py

```diff
@@ -1601,6 +1603,8 @@ def create_tax_cash_basis_entry(self, percentage_before_rec):
                             if line.account_id.reconcile:
                                 #setting the account to allow reconciliation will help to fix rounding errors
                                 to_clear_aml |= line
+                                if percentage_after < 1.0:
+                                    to_clear_aml = to_clear_aml.with_context(skip_full_reconcile_check=True)
```

Comment:
@kebeclibre 

I am highly concerned with the wired 1.0 percent value, can this be linked to something like units of the currency rounding?

Regards

Reply:
@hbto 
percentage is a plain float, without unit.
It represents the proportion of a move being reconcilied
So logically, when the proportion is not one we are not supposed even to to check whether the reconciliation will be full.
However, I definitely admit I don't like the fix :D

Reply:
@kebeclibre When I refer to units of rounding Currency I mean, 

X times the value of the rounding of the currency of company.

regards